packages/babel-plugin-proposal-import-wasm-source/test/fixtures/browser-and-node-esm/import-meta-resolve-fallback-for-browser/output.mjs

```diff
@@ -0,0 +1 @@
+const s = await (typeof process === "object" && process.versions.node ? import("fs").then(fs => fs.promises.readFile(new URL(import.meta.resolve("./x.wasm")))).then(WebAssembly.compile) : WebAssembly.compileStreaming(fetch(import.meta.resolve?.("./x.wasm") ?? new URL("./x.wasm", import.meta.url))));
```

Comment:
`typeof (process?.versions) === "object"`
Would this be better?
Sometimes people use fake `process.env`.
Anyway, no matter how the generated code already contains `?.`.

Reply:
`typeof process?.versions?.node === "string"`

Or this. :

Reply:
No, `process?.version` throws in strict mode if `process` is not defined. The only way to check if a variable exists is to use `typeof thatVariable` directly.

Reply:
Thank you, I understand!